docs/how-to-guides/manage-runs.md

```diff
@@ -45,6 +47,37 @@ column_mapping:
 run: <existing-flow-run-name>
 ```
 
+### Column mapping
+
+Column mapping is a mapping from flow input name to specified values.
+If specified, the flow will be executed with provided value for specified inputs.
+The following types of values in column mapping are supported:
+
+- `${data.<column_name>}` to reference from your test dataset.
+- `${run.outputs.<output_name>}` to reference from your flow output.
+- `STATIC_VALUE` to create static value for all lines for specified column.
+
+### Flow inputs override priority
+
+Flow input value are override based on the following priority: "specified in column mapping" > "default value" > "same name column in provided data".
+
+For example, the following flow will use:
+
+- value "val1" specified by `column_mapping` for input "key1"
+- value "val2" specified in flow's default value for input "key2"
```

Comment:
show the flow yaml at least

Reply:
Added flow YAML and moved to separate file.